bubblewrap.c

```diff
@@ -1327,6 +1333,10 @@ main (int    argc,
   if (!is_privileged)
     opt_unshare_user = TRUE;
 
+  if (opt_unshare_user_try &&
+      stat ("/proc/self/ns/user", &sbuf) == 0)
```

Comment:
What I could see is bubblewrap carrying distro-specific code to detect userns enablement state.  For Debian looking at the sysctl is probably easy, for RHEL...I'm not sure how easy it is to parse module params.

It seems the real fix is for `--unshare-user-try` to detect `EINVAL` from `CLONE_NEWUSER`.  I'm not sure how easy that would be though.